resources/tests.js

```diff
@@ -578,3 +578,55 @@ Suites.push({
         }),
     ]
 });
+
+var stockchartsTests = [
```

Comment:
This needs to be duplicated for each suite, or else the test runner is gonna get confused.